src/systemctl/systemctl.c

```diff
@@ -4303,6 +4333,26 @@ static void print_status_info(
                         printf("n/a\n");
         }
 
+        if (!strv_isempty(i->triggers)) {
+            char **trigger;
+            UnitActiveState state;
+            bool found = false;
+            off = ansi_normal();
+
+            STRV_FOREACH(trigger, i->triggers) {
+                    r = get_state_one_unit(bus, *trigger, &state);
+                    if (r < 0) {
+                            continue;
+                    }
```

Comment:
no {} around single line if blocks

That said, I'd just ignore the error here. i.e. continue and show as is